cli.js

```diff
@@ -8,30 +8,39 @@ const cli = meow(`
 	  github-markdown-css > <filename>
 
 	Options
-	  --light     Light theme
-	  --dark      Dark theme
-	  --list, -l  List available themes
+	  --type      Theme name: 'light', 'dark', 'auto' or other --list values.
+	  --list      List available themes
 
 	Examples
-	  $ github-markdown-css -l
-	  light, dark, dark_dimmed, dark_high_contrast, dark_colorblind, light_colorblind
+	  $ github-markdown-css --list
+	  light
+	  dark
+	  dark_dimmed
+	  dark_high_contrast
+	  dark_colorblind
+	  light_colorblind
 
```

Comment:
We should explain somewhere what `auto` implies.

Reply:
I'll add that to the description of `--type`.